lib/div-alg.gi

```diff
@@ -394,7 +394,9 @@ if (l=4) then
   g:=f/[a^A[3],b^A[4][1]*a^(-A[4][3]),b^(-1)*a*b*a^(-A[4][2])];
 fi;
 
-I:=IsomorphismSpecialPcGroup(g);
+##### Change to Permutation Group for performance ####
```

Comment:
This commit seems the only change that should be here - was the problem that you submitted it into master, not into divalg branch?

Reply:
Same change in two places.  When I posted it on github I had the same problem as two days ago my computer said it was based on github with only one commit ahead of divalg but when I pushed it to github its page showed it based on master with thirty something commits.  So I just closed and deleted it.  I am trying to push a pr/73 using git bash now but its not letting me push as there is no existing pr/73 there yet.

Reply:
That was easy to fix - I restored closed branch, reopened this PR, and changed its base branch - so now it looks like being submitted to divalg, where it should go.